packages/material-ui-utils/src/isMuiElement.d.ts

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { DistributiveOmit } from '@material-ui/types';
```

Comment:
`@material-ui/types` isn't declared as a dependency of the package. It creates this fail: https://github.com/mui-org/material-ui-x/pull/1046. Solved in https://github.com/mui-org/material-ui/pull/24936